src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -381,6 +385,164 @@ public Task GetMigratedServerUrls()
+                    throw new ArgumentException("DatabaseTypeName cannot be null or empty");
+
+                if (string.IsNullOrWhiteSpace(migrationConfiguration.FullPathToMigrator))
+                    throw new ArgumentException("FullPathToMigrator cannot be null or empty");
+
+                if (migrationConfiguration.InputConfiguration == null)
+                    throw new ArgumentException("InputConfiguration cannot be null");
+
+                if (Directory.Exists(migrationConfiguration.FullPathToMigrator) == false)
+                    throw new InvalidOperationException($"Directory {migrationConfiguration.FullPathToMigrator} doesn't exist");
+
+                if (migrationConfiguration.InputConfiguration.TryGet("Command", out string command) == false)
+                    throw new ArgumentException("Cannot find the Command property in the InputConfiguration");
+
+                const string migratorFileName = "Raven.Migrator.dll";
+                var path = Path.Combine(migrationConfiguration.FullPathToMigrator, migratorFileName);
+                if (File.Exists(path) == false)
+                    throw new InvalidOperationException($"The file '{migratorFileName}' doesn't exist in path: {migrationConfiguration.FullPathToMigrator}");
+
+                var processStartInfo = new ProcessStartInfo
+                {
+                    FileName = "dotnet",
+                    Arguments = $"{migratorFileName} {migrationConfiguration.DatabaseTypeName}",
+                    WorkingDirectory = migrationConfiguration.FullPathToMigrator,
+                    CreateNoWindow = true,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    RedirectStandardInput = true,
+                    UseShellExecute = false
+                };
+
+                Process process = null;
+                try
+                {
+                    process = Process.Start(processStartInfo);
+                }
+                catch (Exception e)
+                {
+                    process?.Kill();
+                    throw new InvalidOperationException("Unable to execute Migrator." + Environment.NewLine +
+                                                        "Command was: " + Environment.NewLine +
+                                                        (processStartInfo.WorkingDirectory ?? Directory.GetCurrentDirectory()) + "> "
+                                                        + processStartInfo.FileName + " " + processStartInfo.Arguments, e);
+                }
+
+                await process.StandardInput.WriteLineAsync(migrationConfiguration.InputConfiguration.ToString());
+
+                var isExportCommand = command == "export";
+                if (isExportCommand == false)
+                {
+                    try
+                    {
+                        var line = process.StandardOutput.ReadLine();
```

Comment:
Here you are reading a _single_ line, is that intentional, what about other output?

Reply:
Intentional. The ouput from tool is here: https://github.com/ravendb/ravendb/blob/1932a9d2799b886aad3040c4192624975fcb3148/tools/Raven.Migrator/MigrationHelpers.cs#L48